### PR TITLE
shared.cfg: Update pci_test_cmd config for windows guest

### PR DESCRIPTION
**Pull request by spcui**
ID: 1204027

### CODE REVIEW

File: shared/cfg/guest-os/Windows.cfg [hunk 1]
```diff
@@ -149,12 +149,45 @@
...
+        pci_test_cmd += " && echo detail disk >> cmd_detail"
+        pci_test_cmd += " && echo exit >> cmd_detail"
+
+        pci_test_cmd += "&& echo find "Status : Online" disk1detail > check_cmd.bat"
+        pci_test_cmd += "&& echo if errorlevel 1 ( diskpart /s cmd_online ) >> check_cmd.bat"
+        pci_test_cmd += "&& echo diskpart /s cmd_cleanreadonly >> check_cmd.bat"
+        pci_test_cmd += "&& echo diskpart /s cmd_mount >> check_cmd.bat"
+        pci_test_cmd += "&& echo diskpart /s cmd_remove >> check_cmd.bat"
+        pci_test_cmd += "&& echo diskpart /s cmd_offline >> check_cmd.bat"
+        pci_test_cmd += "&& echo if errorlevel 1 >> exit 1"
```
Comment (xutian) [inline shared/cfg/guest-os/Windows.cfg]: This line looks has error !!!

Comment (spcui) [inline shared/cfg/guest-os/Windows.cfg]: @xutian already tested this patch, not found error info, the result is passed.

**Comment by spcui** [comment]
# ../../tools/scan_results.py 
Test                                                                                                                                                                         Status     Seconds  Info
----                                                                                                                                                                         ------     -------  ----
        (Result file: /root/staf-kvm-devel/autotest-devel/client/results/default/status)
virt.qemu.smp_8.8192m.repeat1.Host_RHEL.6.5.spice.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_qcow2      GOOD       214      completed successfully
virt.qemu.smp_8.8192m.repeat1.Host_RHEL.6.5.spice.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_raw        GOOD       159      completed successfully
virt.qemu.smp_8.8192m.repeat1.Host_RHEL.6.5.spice.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.block_hotplug.block_virtio.multi_pci.fmt_qcow2    GOOD       201      completed successfully
virt.qemu.smp_8.8192m.repeat1.Host_RHEL.6.5.spice.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.block_hotplug.block_virtio.multi_pci.fmt_raw      GOOD       191      completed successfully
----                                                                                                                                                                         GOOD       773

**Comment by CongSmile** [comment]
Hi @spcui 

Test failed with my test.
my cfg: win8.x86_64.0 and virtio_scsi

        START   virt.qemu.smp_4.16384m.repeat1.run_test.Host_RHEL.7.2.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_qcow2  virt.qemu.smp_4.16384m.repeat1.run_test.Host_RHEL.7.2.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_qcow2  timestamp=1434449393    timeout=14400   localtime=Jun 16 18:09:53
                FAIL    virt.qemu.smp_4.16384m.repeat1.run_test.Host_RHEL.7.2.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_qcow2  virt.qemu.smp_4.16384m.repeat1.run_test.Host_RHEL.7.2.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_qcow2  timestamp=1434450394    localtime=Jun 16 18:26:34       Check for block device failed after PCI hotplug. Output: '1 \n\nC:\\>find "Status : Online" disk1detail  \n\n---------- DISK1DETAIL\n\nC:\\>if errorlevel 1 (diskpart /s cmd_online  ) \n\nMicrosoft DiskPart version 6.2.9200\n\nCopyright (C) 1999-2012 Microsoft Corporation.\nOn computer: WIN-57TVDHA9ADJ\n\nDisk 1 is now the selected disk.\n\nDiskPart successfully onlined the selected disk.\n\nLeaving DiskPart...\n\nC:\\>diskpart /s cmd_cleanreadonly  \n\nMicrosoft DiskPart version 6.2.9200\n\nCopyright (C) 1999-2012 Microsoft Corporation.\nOn computer: WIN-57TVDHA9ADJ\n\nDisk 1 is now the selected disk.\n\nDisk attributes cleared successfully.\n\nLeaving DiskPart...\n\nC:\\>diskpart /s cmd_mount  \n\nMicrosoft DiskPart version 6.2.9200\n\nCopyright (C) 1999-2012 Microsoft Corporation.\nOn computer: WIN-57TVDHA9ADJ\n\nDisk 1 is now the selected disk.\n\nVirtual Disk Service error:\nThere is not enough usable space for this operation.\n\n\nC:\\>diskpart /s cmd_remove  \n\nMicrosoft DiskPart version 6.2.9200\n\nCopyright (C) 1999-2012 Microsoft Corporation.\nOn computer: WIN-57TVDHA9ADJ\n'    [context: Start checking new added device]
        END FAIL        virt.qemu.smp_4.16384m.repeat1.run_test.Host_RHEL.7.2.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_qcow2  virt.qemu.smp_4.16384m.repeat1.run_test.Host_RHEL.7.2.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_qcow2  timestamp=1434450394    localtime=Jun 16 18:26:34

session log:
2015-06-16 18:25:20: C:\>diskpart /s cmd_offline
2015-06-16 18:25:20:
2015-06-16 18:25:20: Microsoft DiskPart version 6.2.9200
2015-06-16 18:25:20:
2015-06-16 18:25:20: Copyright (C) 1999-2012 Microsoft Corporation.
2015-06-16 18:25:20: On computer: WIN-57TVDHA9ADJ
2015-06-16 18:25:28:
2015-06-16 18:25:28: Disk 1 is now the selected disk.
2015-06-16 18:25:28:
2015-06-16 18:25:28: DiskPart has encountered an error: The system cannot find the file specified.
2015-06-16 18:25:28: See the System Event Log for more information.
2015-06-16 18:25:28:
2015-06-16 18:25:28: C:\>


Could you check it again?

Thanks.

**Comment by spcui** [comment]
Hi @CongSmile still passed during my testing. 
# ../../tools/scan_results.py 
Test                                                                                                                                                                  Status     Seconds  Info
----                                                                                                                                                                  ------     -------  ----
        (Result file: /root/staf-kvm-devel/autotest-devel/client/results/default/status)
virt.qemu.smp_8.8192m.repeat1.Host_RHEL.6.5.spice.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_qcow2    GOOD       292      completed successfully
----                                                                                                                                                                  GOOD       300

**Comment by CongSmile** [comment]
Could you test it on RHEL7 host?

And I have pasted the error info in session log, could you have a look at it?

Thanks.

**Comment by spcui** [comment]
Hi @CongSmile 

# ./tools/scan_results.py 
Test                                                                                                                                                                           Status     Seconds  Info
----                                                                                                                                                                           ------     -------  ----
        (Result file: /root/staf-kvm-devel/autotest-devel/client/results/default/status)
virt.qemu.smp_4.8192m.repeat1.run_test.Host_RHEL.7.1.spice.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_hotplug.block_virtio.one_pci.fmt_qcow2    GOOD       343      completed successfully

**Comment by xutian** [comment]
Frankly speaking. I don't like this fix. I hope to write a tool to do it.

**Comment by spcui** [comment]
Close it, will update with xutian's comment.

**Pull request closed without merging**